express/blocks/search-marquee/search-marquee.js

```diff
@@ -0,0 +1,368 @@
+import { getLibs } from '../../scripts/utils.js';
+import { decorateButtonsDeprecated, addTempWrapperDeprecated } from '../../scripts/utils/decorate.js';
+import { getIconElement } from '../../scripts/utils/icons.js';
+import { buildFreePlanWidget } from '../../scripts/widgets/free-plan.js';
+import buildCarousel from '../../scripts/widgets/carousel.js';
+import fetchAllTemplatesMetadata from '../../scripts/utils/all-templates-metadata.js';
+import BlockMediator from '../../scripts/block-mediator.min.js';
+
+const imports = await
+Promise.all([import(`${getLibs()}/features/placeholders.js`),
+  await import(`${getLibs()}/utils/utils.js`),
+  await import(`${getLibs()}/utils/samplerum.js`)]);
+const { replaceKey } = imports[0];
+const { createTag, getConfig, getMetadata } = imports[1];
+const { sampleRUM } = imports[2];
```

Comment:
sampleRum could be a dynamic import though ;) . I know the code is not originally from you, but we can make the imports better at least while migrating.